index.html

```diff
@@ -262,6 +273,138 @@ <h2 id="civname">Loading…</h2>
 
     }
 
+    /**
+     * Create the Cross-Reference badges. This is done at load time in order to avoid re-making the
+     * badges at runtime per-click on a new unit.
+     * 
+     * @return A container with buttons + images for each civ to be used in cross referencing.
+     */
+     function createXRefBadges() {
+        let xRefLinks = document.createElement('div');
+        xRefLinks.id = "helptext__x_ref__container";
+
+        for (let civ of Object.keys(data.civs)) {
+            let xRefLink = document.createElement('button');
+            xRefLink.addEventListener('click', function() {
+              document.getElementById('civselect').value=civ;
+              loadCiv();
+            });
+
+            // Add a badge for this civ if this element isn't disabled, or if it is a unique unit and not enabled.
+            let xRefImage = document.createElement('img');
+
+            xRefImage.src = `./img/Civs/${civ.toLowerCase()}.png`;
+            xRefImage.title = `${civ}`;
+            xRefImage.id = `xRef__badge__${civ}`;
+            xRefLink.appendChild(xRefImage);
+            xRefLinks.appendChild(xRefLink);
+        }
+        return xRefLinks;
+    }
+
+    /** 
+     * Set on/off of all cross reference badges for a single unit.
+     * 
+     * @param {string} name The name of the entity being cross-reference. 
+     * @param {string} type The type of the entity being cross-reference.
+     */
+    function badgeXRef(name, type) {
```

Comment:
I am not sure if most comments in this function really add too much value or if they can be removed (except the "There are 2 ways for a UNIQUEUNIT to be disabled." etc. comments)